inksdk/inksample/src/main/java/com/microsoft/device/inksample/MainActivity.kt

```diff
@@ -166,24 +207,24 @@ class MainActivity : AppCompatActivity() {
      */
     inner class RainbowPaintHandler : DynamicPaintHandler {
         // frequency = 2*Math.PI/steps
-        var frequency = .3
-        var steps = 20
-        var i = 0
+        private var frequency = .3
+        private var steps = 20
+        private var i = 0
         override fun generatePaintFromPenInfo(penInfo: InputManager.PenInfo): Paint {
-            var paint = Paint()
+            val paint = Paint()
 
             if (i > steps) i = 0 else i++
             paint.color = Color.argb(
                 255,
-                (Math.sin(frequency * i + 0) * 127 + 128).toInt(),
-                (Math.sin(frequency * i + 2) * 127 + 128).toInt(),
-                (Math.sin(frequency * i + 4) * 127 + 128).toInt()
+                (sin(frequency * i + 0) * 127 + 128).toInt(),
```

Comment:
Although this is for the sample code, I think is good if these values are also in constant variables with a meaningful name so others can understand better what's going on there.